gulp-tasks/utils/rollup-helper.js

```diff
@@ -47,16 +51,18 @@ module.exports = {
             // private variable and names that start with a single underscore
             // followed by a letter. This restriction to avoid mangling
             // unintentional fields in our or other libraries code.
-            regex: /^_[A-Za-z]/,
+            regex: /(?:^_[A-Za-z])|(?:babelHelpers)|(?:asyncToGenerator)/,
             // If you are getting an error due to a property mangle
             // set this flag to true and the property will be changed
             // from '_foo' to '$_foo$' to help diagnose the problem.
             debug: false,
           },
         },
+        nameCache: global.uglifyNameCache,
```

Comment:
Nit: does this need to be on the global scope? Could we require the ulgify helper and reference an object in the module?